dotCMS/src/main/java/com/dotcms/util/SecurityUtils.java

```diff
@@ -77,28 +76,6 @@ public static String stripReferer ( HttpServletRequest request, String referer )
 
                 String serverName = request.getServerName();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "serverName". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1854)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "serverName" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1481)